docs/docs/code_submission.md

```diff
@@ -32,20 +31,19 @@ The folder structure above does contain some code but it is not immediately
 obvious which team does it belong to.
 
 To make it obvious, please create a file called `team_name.txt` on the same
-level as `robot1/`, `robot2/` and `robot3/`. This file should contain a single
+level as `team_name/`. This file should contain a single
```

Comment:
Same level as 'team_name' or same level as 'robot/'?

Reply:
`team_name/` is meant to be `robot/`. I'll change the wording to `robot/`. At first I thought that participants will already submit the controller with unique name.

@mrshu are we going to rename directories based on team name to make then unique?

Reply:
@Adman At this point I believe even using folder names like `001` like @dbscoach did in the Demo event ought to work. Or did I get it wrong?

Reply:
@mrshu yeah, but renaming the folder and controller will be our job, so we do not have to bother participants with folder names.

Reply:
@Adman true, that I completely agree with